src/utility/commonKeys.js

```diff
@@ -0,0 +1,16 @@
+// keep consistent with codemirror:
+// https://github.com/codemirror/CodeMirror/blob/ed8dfeb5e2/src/util/browser.js#L17
+const isMacOs = /Mac OS X 1\d\D([8-9]|\d\d)\D/.test(navigator.userAgent);
+
+const commonKeys = {
+  // Persistent search box in Query Editor
+  [isMacOs ? "Cmd-F" : "Ctrl-F"]: "findPersistent",
```

Comment:
The previous behavior is definitely clowny. However, I don't quite trust the codemirror regex, since it only returns true for mountain lion and higher. Let's keep the former way for now.

Reply:
Let's use `navigator.platform` - much less clowny :)

Reply:
this UA detection seems needless here. Just make two entries, one for Cmd-F and one for Ctrl-F, with the same value. Thats the usual codemirror keymap MO